Scenes/Popup_button.gd

```diff
@@ -0,0 +1,9 @@
+extends Node
```

Comment:
Jeśli pracujesz z Node'ami typu control, to niech Root w scenie też będzie Node'em typu Control. Później mogą wyjść dziwne bugi w projektcie.